pkg/virt-operator/kubevirt.go

```diff
@@ -511,6 +511,36 @@ func NewKubeVirtController(
 		return nil, err
 	}
 
+	_, err = c.informers.ClusterInstancetype.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			c.genericAddHandler(obj, nil)
```

Comment:
Do we need to add an expectations for cluster instancetype and preference?
```
	ClusterInstancetype *controller.UIDTrackingControllerExpectations
	ClusterPreference     *controller.UIDTrackingControllerExpectations
```
@xpivarc

Reply:
`expectations` are optimization, I don't we need them